Server/Content/Sources/App/Migrations/CreateText.swift

```diff
@@ -0,0 +1,16 @@
+import Fluent
+
+struct CreateText: Migration {
+    func prepare(on database: Database) -> EventLoopFuture<Void> {
+        return database.schema("texts")
+            .id()
+            .field("title", .string, .required)
+            .field("text", .string, .required)
+            .field("backward_text", .string)
+            .create()
+    }
+
+    func revert(on database: Database) -> EventLoopFuture<Void> {
+        return database.schema("texts").delete()
```

Comment:
You can replace hardcoded string with `Text.schema` (because that variable already contains this string)